test-app/runtime/src/main/java/com/tns/Runtime.java

```diff
@@ -490,7 +490,8 @@ private void init(Logger logger, String appName, String nativeLibDir, File rootD
                 throw new RuntimeException("Fail to initialize Require class", ex);
             }
 
-            initNativeScript(getRuntimeId(), Module.getApplicationFilesPath(), nativeLibDir, logger.isEnabled(), isDebuggable, appName, appConfig.getAsArray(), callingJsDir, appConfig.getMaxLogcatObjectSize());
+            initNativeScript(getRuntimeId(), Module.getApplicationFilesPath(), nativeLibDir, logger.isEnabled(), isDebuggable, appName, appConfig.getAsArray(), callingJsDir, appConfig.getMaxLogcatObjectSize(),
+                appConfig.getForceLog() || "timeline".equalsIgnoreCase(appConfig.getProfilingMode()));
```

Comment:
Just for readability you can move this `appConfig.getForceLog() || "timeline".equalsIgnoreCase(appConfig.getProfilingMode())` expression to a local variable.